packages/styles/scss/components/_socialmedia.scss

```diff
@@ -9,10 +9,7 @@
   $icon-size: $spacing-spacing-4;
 
   @mixin get-icon($icon, $theme: $default-theme) {
-    @include dataurlicon(
-      $icon,
-      map-get($color, "socialmedia", $theme, "icon", "color")
-    );
+    @include dataurlicon($icon, rgba(30, 45, 190, 1));
```

Comment:
Hey team since we are moving away from map-get and using css variables, there a small problem in this one. Css variables cant be used in mixins. 

For the time being I used `rgba(30, 45, 190, 1)` as it doesn't support hex values as well. 

How should we go with this one @justintemps @bashlk @GGKapanadze